module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -674,7 +674,12 @@ protected function createRequestList($requestType)
      */
     public function getStatus($id)
     {
-        return $this->getSimulatedStatus($id);
+        $status = $this->getSimulatedStatus($id);
+        foreach (array_keys($status) as $i) {
```

Comment:
Something to consider: at present, the various notes fields we're displaying are only part of the ILS driver spec for the `getHolding` method. However, `GetItemStatus` uses the `getStatus` method instead. Many, but not all, ILS drivers make `getStatus` a proxy for `getHolding`, but in some cases, `getStatus` returns fewer fields in order to be more performant for multi-item lookups. When we merge this, we should probably add a note to the ILS driver spec (and maybe to the config.ini comments) to acknowledge this situation.

In any case, for testing purposes, I added the note fields to the demo driver here so we can see things in action without needing a real ILS.